File: web3rpc/rpc-specs/code-samples/javascript/debug/runtimeTracing/goTrace.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const file = "go.trace";
  const seconds = 5;

  provider.debug
    .goTrace(file, seconds, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/debug/runtimeTracing/goTrace.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

fileName = "go.trace"
seconds = 5

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.go_trace(fileName, seconds)

print(debug_response)
